src/ls/LS.mli

```diff
@@ -0,0 +1 @@
+val start : Runner_config.t -> unit
```

Comment:
comment

Reply:
Can be "Entry point of the language server. This will start the server following the LSP protocol and listen on port XXX for requests from VScode or other LSP clients.